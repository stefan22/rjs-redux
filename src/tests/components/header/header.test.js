import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from '../../../components/header/header';


describe('Header component', () => {
  it('should render header component correctly', () => {
    const wrapper = shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
    expect(wrapper.find('.header').length).toBe(1);
  });

  it('should have a class name of `header`', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('.header').length).toBe(1);
  });

});

