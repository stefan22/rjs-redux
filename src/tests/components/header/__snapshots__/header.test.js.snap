// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render header component correctly 1`] = `
<div
  className="header"
>
  <h1
    className="text-center border-bottom"
  />
  <p
    className="text-center"
  />
</div>
`;
